R/model_importance.R

```diff
@@ -134,6 +139,16 @@ model_importance <- function(forecast_data,
     min(forecast_date_list), max(forecast_date_list), length(forecast_date_list)
   ))
 
+  # model ids
+  model_id_list <- unique(valid_tbl$model_id)
+
+  # Give a message for the user to check the model IDs
+  message(paste(
+    "The available model IDs are:\n",
+    paste("\t", model_id_list, collapse = "\n"),
+    "\n(a total of", length(model_id_list), "models)\n"
+  ))
+
```

Comment:
consider adding some pseudo-code or prototype code (in comments is ok, if lintr is complaining) to show what the eventual structure of this function might look like.